### ISSUE TITLE
etcd3 v0.2.1 packaging failure

### ISSUE THREAD

**Issue by akihiro** [mixer/etcd3#23]
I try use etcd3 v0.2.1. but etcd3 package cannot be installed.

client.patch.js is missing in package?

```console
$ npm i etcd3

> grpc@1.3.8 install /home/akihiro/test-etcd3/node_modules/grpc
> node-pre-gyp install --fallback-to-build --library=static_library

[grpc] Success: "/home/akihiro/test-etcd3/node_modules/grpc/src/node/extension_binary/grpc_node.node" is installed via remote

> etcd3@0.2.1 install /path/to/node_modules/etcd3
> node client.patch.js

module.js:471
    throw err;
    ^

Error: Cannot find module '/home/akihiro/test-etcd3/node_modules/etcd3/client.patch.js'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:393:7)
    at startup (bootstrap_node.js:150:9)
    at bootstrap_node.js:508:3
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm ERR! Linux 4.9.0-3-amd64
npm ERR! argv "/home/akihiro/.nvm/versions/node/v6.10.1/bin/node" "/home/akihiro/.nvm/versions/node/v6.10.1/bin/npm" "i" "etcd3"
npm ERR! node v6.10.1
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE

npm ERR! etcd3@0.2.1 install: `node client.patch.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the etcd3@0.2.1 install script 'node client.patch.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the etcd3 package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node client.patch.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs etcd3
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls etcd3
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/akihiro/test-etcd3/npm-debug.log
```

**Comment by drewfish** [comment]
Yeah, I get that too with a fresh git clone:
* clone, cd into directory
* `npm install`
* `npm run build:ts`
* `node --eval 'Error.stackTraceLimit=50; require(".")'`

It shows this error:
```
/Users/folta/tmp/etcd3/node_modules/grpc/src/node/index.js:141
      builder = ProtoBuf.loadProtoFile(filename);
                         ^

TypeError: ProtoBuf.loadProtoFile is not a function
    at Object.load (/Users/folta/tmp/etcd3/node_modules/grpc/src/node/index.js:141:26)
    at Object.<anonymous> (/Users/folta/tmp/etcd3/lib/src/connection-pool.js:8:23)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/folta/tmp/etcd3/lib/src/index.js:7:27)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at [eval]:1:27
    at ContextifyScript.Script.runInThisContext (vm.js:25:33)
    at Object.runInThisContext (vm.js:97:38)
    at Object.<anonymous> ([eval]-wrapper:6:22)
    at Module._compile (module.js:570:32)
    at evalScript (bootstrap_node.js:354:27)
    at run (bootstrap_node.js:123:11)
    at run (bootstrap_node.js:390:7)
    at startup (bootstrap_node.js:122:9)
    at bootstrap_node.js:505:3
```

**Comment by connor4312** [comment]
Okay, thanks for the reports. I looks like grpc changed under us and the patch is no longer effective (and regardless should no longer be necessary with https://github.com/grpc/grpc/pull/11515). I've unpublished 0.2.1, I'll revisit this tomorrow.

**Comment by connor4312** [comment]
Sorry for the delay, there were some other test failures I needed to address.

I've published 0.2.2 which resolves these issues. Please let me know if you hit any more problems!
